learning_observer/learning_observer/canvas.py

```diff
@@ -0,0 +1,308 @@
+import os
```

Comment:
Okay now I'm a bit more confused. Is the `learning_tools/` directory being used at all? If not, that's okay, but it needs to be removed.